### PR TITLE
SUP-3653: add and/or edit prefix values for US Sens

### PR DESCRIPTION
**Pull request by OdeclasV**
Add and edit prefix options; edit step requirement

### CODE REVIEW

File: members/C000127.yaml [hunk 1]
```diff
@@ -76,12 +76,10 @@ contact_form:
           Ms.: Ms.
           Miss: Miss
           Mr. and Mrs.: Mr. and Mrs.
-          Proffesor: Proffesor
+          Profesor: Profesor
```
Comment (tomdionesotes) [inline members/C000127.yaml]: misspelled on their site?

File: members/M001176.yaml [hunk 2]
```diff
@@ -67,15 +67,6 @@ contact_form:
           commands: [ "elements[0].value = elements[0].value.replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
           required: true
     - select:
-        - name: Your Prefix
```
Comment (tomdionesotes) [inline members/M001176.yaml]: not added back as a `select`?

File: members/C000127.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/C000127.yaml]: I think it was at some point, but it's now fixed.

File: members/M001176.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/M001176.yaml]: Nah, it's completely missing from the form https://www.merkley.senate.gov/contact/

They might still add it back later tho, as I saw that the validation for `prefix` is still present on the page

Comment (tomdionesotes) [reply members/M001176.yaml]: As long as the form submits with the update yaml, it's all good!

File: members/C000127.yaml (hunk 1 above)
Comment (tomdionesotes) [reply members/C000127.yaml]: This should be Professor (two s's)

File: members/M001176.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/M001176.yaml]: oh yeah, my bad should've mentioned that. I did a test with the updated yaml and it succeeded. Thanks for checking

File: members/C000127.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/C000127.yaml]: http://gph.is/1W2P1V1

**Pull request merged**